### ISSUE TITLE
Seafile Client 6.1 improperly attempting to connect to 127.0.0.1

### ISSUE THREAD

**Issue by jeffmikels** [haiwen/seafile#1937]
Screenshot explains it all, but my client is attempting to connect to a seafile server on the localhost, and since I don't have a server on my localhost, it is failing and displaying a little red triangle at the bottom of the seafile client window.

Can anyone help me fix this?

![screen shot 2017-08-02 at 4 04 24 pm](https://user-images.githubusercontent.com/3506020/28892630-a7f203ba-779c-11e7-878f-120ecf76b933.png)

**Comment by flobiree** [comment]
Same problem here, no idea…

**Comment by sjcjonker** [comment]
I assume it's a server matter, i.e. the server sharing the wrong avatar url. However in any case happy to provide logging. But please inform what to share.

This is the error in applet.log:
`[10/23/17 07:21:25][api] network error for http://127.0.0.1:8000/media/avatars/9/b/a2b7757d3ab47dddb182a252574f78/resized/80/b7d6a6ee6a216389271559d5d7380e93_Yhxn8rM.png: Connection refused`

Which seems an untranslated URL between the webserver and the seafile backend, as on the server it does exist and provides an 200 OK:

```
[user@server ~]$ curl -v -o image.png http://127.0.0.1:8000/media/avatars/9/b/a2b7757d3ab47dddb182a252574f78/resized/80/b7d6a6ee6a216389271559d5d7380e93_Yhxn8rM.png
* About to connect() to 127.0.0.1 port 8000 (#0)
*   Trying 127.0.0.1...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Connected to 127.0.0.1 (127.0.0.1) port 8000 (#0)
> GET /media/avatars/9/b/a2b7757d3ab47dddb182a252574f78/resized/80/b7d6a6ee6a216389271559d5d7380e93_Yhxn8rM.png HTTP/1.1
> User-Agent: curl/7.29.0
> Host: 127.0.0.1:8000
> Accept: */*
>
< HTTP/1.1 200 OK
< Server: gunicorn/19.4.5
< Date: Mon, 23 Oct 2017 05:35:54 GMT
< Connection: keep-alive
< Last-Modified: Tue, 10 Oct 2017 20:43:21 GMT
< Content-Length: 16798
< Content-Type: image/png
< Content-Language: en
< Vary: Accept-Language, Cookie
<
{ [data not shown]
100 16798  100 16798    0     0  1371k      0 --:--:-- --:--:-- --:--:-- 1491k
* Connection #0 to host 127.0.0.1 left intact
```

**Comment by Jasper-Ben** [comment]
Seams to be related to the avatar thing. Experiencing the same here.

Seahub log:
`[ERROR] seahub.thumbnail.views:77 latest_entry [Errno 2] No such file or directory: '/home/seafile/seafile-server-6.1.2/seahub/seahub/../../../seahub-data/thumbnail/1024/019796d67dc7645ff612c66ff8dcc8338669c501'`

Any advise on how to clean this up?

**Comment by lilarcor** [comment]
i experienced the same issue. File syncing is fine so far.

**Comment by LambertCliff** [comment]
Same here. No idea why. File syncing works also perfect.

**Comment by shoeper** [comment]
According to the changelog this has been fixed in 6.1.4 (2017/12/20)

**Comment by Jasper-Ben** [comment]
Can confirm that this is fixed on my side. Thanks.
